README.md

```diff
@@ -35,10 +34,17 @@ go get -u github.com/haya14busa/gopkgs/cmd/gopkgs
 ```
 $ gopkgs -h
 Usage of gopkgs:
-  -fullpath
-        output absolute file path to package directory. ("/usr/lib/go/src/net/http")
-  -short
-        output vendorless import path ("net/http", "foo/bar/vendor/a/b")
+  -f string
+    	output format of the package (default "{{.ImportPathShort}}")
```

Comment:
Default should be "ImportPath" not to break backward compatibility.

Reply:
The existing behavior is `-short=true` equal to `ImportPathShort`